src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -21,6 +21,8 @@
 
     <!--This property sets the default output types supported by the project system.-->
     <SupportedOutputTypes Condition="'$(SupportedOutputTypes)' == ''">Exe;WinExe;Library</SupportedOutputTypes>
+
+    <SuppressOutOfDateMessageOnBuild>true</SuppressOutOfDateMessageOnBuild>
```

Comment:
We always check if it's not already set - this enables flavors, SDKs, etc to overrule what we are setting here.